vscode-sbcl.md

```diff
@@ -0,0 +1,190 @@
+added to your SBCL config file per [these instructions](getting-started.html#install-quicklisp)
+
+- [VsCode](https://code.visualstudio.com) with [command
+  line](https://code.visualstudio.com/docs/setup/mac#_launching-from-the-command-line)
+  installed
+- [Alive](https://marketplace.visualstudio.com/items?itemName=rheller.alive)
+  Common Lisp plugin for VsCode
+- [SBCL](http://www.sbcl.org)
+- [Quicklisp](https://www.quicklisp.org/beta/)
+
+### Introduction: Connect VSCode to a REPL
+
+1. Inside of VSCode, open a lisp file that you want to edit.
+   - If you
+   don't have one, just create a new one called `hello.lisp`
+2. Inside of VSCode, open the Command Palette on the menu at the top
+   where it says `View/Command Palette` and start an instance of SBCL
+   running a Swank server attached to your VSCode REPL by choosing:
+   `Alive: Start REPL And Attach`.
+   - You will see a small window pop up that says `REPL Connected`
+
+Congrats, You now have a VSCode instance running a REPL attached to a
+Swank server running on port 4005 of a running SBCL image.  You can
+now evaluate statements in your file and they will be processed in
+your running SBCL instance.
+
+_To disconnect your REPL and shut down your SBCL instance, open the
+Command Palette on the menu at the top where it says `View/Command
+Palette` and choose: `Alive: Detach from REPL`_
+
+There are keybindings for every operation, feel free to explore and
+modify those as needed.
+
+### Recipes
+
+_All recipes assume you have a file open in VSCode running with an attached
+REPL unless otherwise stated._
+
+_When evaluating an expression, you choose the expression to evaluate
+by putting your cursor anywhere in or immediately following the expression._
+
+#### Evaluate a statement in-line
+
+1. In your open file in your editor window, enter:
+   ~~~lisp
+   (+ 2 2)
+   ~~~
+2. Open the Command Palette on the menu at the top `View/Command
+   Palette` and choose `Alive: Inline Eval`
+3. You will see a small pop up that says `=> 4 (3 bits, #x4, #o4,
+   #b100)`, which is the result
+
+   _Evaluating a statement in-line is exactly the same as sending it to
+   the REPL.  The only difference is how it is displayed._
+
+#### Evaluate a statement
+
+1. In your open file in your editor window, enter:
+   ~~~lisp
+   (+ 2 2)
+   ~~~
+2. Open the Command Palette on the menu at the top `View/Command
+   Palette` and choose `Alive: Send To REPL`
+3. You will see the expression show up in the REPL along with the result.
+   ~~~lisp
+   CL-USER>
+   (+ 2 2)
+   4
+   CL-USER>
+   ~~~
+
+#### Use the Interactive Debugger to abort
+
+1. In your open file in your editor window, enter:
+   ~~~lisp
+   (defun divide (x y)
+     (/ x y))
+   ~~~
+2. Put your cursor after the last parenthesis if it isn't already there.
+   Open the Command Palette on the menu at the top `View/Command
+   Palette` and choose `Alive: Inline Eval` to load your `define`
```

Comment:
Does it have a "compile" equivalent ? In SLIME there is a difference (https://lispcookbook.github.io/cl-cookbook/emacs-ide.html#evaluating-and-compiling-lisp-in-slime)

Reply:
It's in-progress.  Alive doesn't have it yet, but it's being worked on.  I will update the docs once it is in there.

Reply:
Ok, it's in there now.  I added it to the docs as well.  Once the PR to alive is in this should be good to merge.